tests/swarmtest/swarmtest.go

```diff
@@ -20,20 +22,27 @@ type SwarmTest struct {
 	NumWorkers int
 	Compile    func() error
 	Run        func() error
+	NumJobs    int
+	Timeout    time.Duration
+	mutex      *sync.Mutex
```

Comment:
What happens without the mutex? I never saw anything bad come of it since we always exit right after killing.